Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -129,7 +129,7 @@
 # Tastes like nachos, cheese.
 
 - type: entity
-  name: Cuban nachos
+  name: cuban nachos
```

Comment:
Cuba is a place, and therefore a proper noun. Proper nouns should remain capitalized.